addons/pt/saft/extensions.go

```diff
@@ -748,6 +748,55 @@ var extensions = []*cbc.Definition{
 					`),
 				},
 			},
+			{
+				Code: "M44",
+				Name: i18n.String{
+					i18n.EN: "Article 6.º of the CIVA – Specific rules",
+					i18n.PT: "Artigo 6.º do CIVA – Regras específicas",
+				},
+				Desc: i18n.String{
+					i18n.EN: here.Doc(`
+						To be used in operations not localized in Portugal due to the
+						exception rules in numbers 7 and following of article 6.º of the
```

Comment:
I think it'd be useful here to try and describe what the "exception rules in numbers 7 and following" actually are. All this description is really saying is the same thing as the title, but longer!